src/assignments/scrabble-scorer.rst

```diff
@@ -0,0 +1,336 @@
+value. The program needs to be rewritten for two reasons:
+
+#. The dictionary used to store the letter point values is inefficient.
+#. The program should have three scoring algorithms and allow the user to pick
+   which one to use.
+
+Your updates will make the program more efficient and user friendly.
+
+**Starter Code** can be found here.  Fork the code then clone it to your machine and work on it in Visual Studio.
+
+Requirements
+------------
+
+.. admonition:: Note
+
+   The requirements below are what your END assignment will look like.
+   This assignment is broken down so you can complete small pieces as you go.
+   You need to move sequentially starting at part A below. Please read the
+   WHOLE assignment page before starting.
+
+#. Create an ``InitialPrompt`` method that asks the user which scoring
+   algorithm to use.
+#. Create a ``Transform`` method that takes in the ``oldPointStructure``
+   dictionary and *returns* a ``newPointStructure`` dictionary.
+#. Code three different scoring functions for assigning points to a word.
+#. Create a ``ScoringAlgorithms`` method to invoke one of the three scoring methods. 
+#. Create a ``RunProgram`` method to serve as the starting point for your
+   program.
+
+Instructions
+-------------
+
+.. admonition:: Note
+
+   As you create your methods, think about where the output is best utilized.  Will it be something a user only needs to read?  
+   Will it become a parameter for another method?  Revisit the :ref:`Methods introduction chapter <methods>` if you feel unsure.
+
+A) Initial Prompt
+^^^^^^^^^^^^^^^^^^
+
+In the starter code, the Scrabble Scorer only uses one scoring algorithm. For
+the new version, we want to let the user pick between three algorithms. Define
+an ``InitialPrompt`` method that introduces the program and asks the user
+which scoring algorithm to use.
+
+Your prompt could look something like:
+
+::
+
+   Welcome to the Scrabble score calculator!
+
+   Which scoring algorithm would you like to use?
+
+   1 - Scrabble: The traditional scoring algorithm.
+   2 - Simple Score: Each letter is worth 1 point.
+   3 - Bonus Vowels: Vowels are worth 3 pts, and consonants are 1 pt.
+
+   Enter 1, 2 or 3:
+
+B) Transform
+^^^^^^^^^^^^^
+
+Currently, the software uses the data structure below for the traditional
+Scrabble scoring algorithm. Take a few moments to review how the
+``oldPointStructure`` dictionary relates a point value to a letter.
+Be sure to notice the data types of this dictionary.
+
+.. sourcecode:: csharp
+   :linenos:
+
+   public static Dictionary<int, string> oldPointStructure = new Dictionary<int, string>()
+   {
+      {1, "A, E, I, O, U, L, N, R, S, T"},
+      {2, "D, G"},
+      {3, "B, C, M, P" },
+      {4, "F, H, V, W, Y" },
+      {5, "K" },
+      {8, "J, X" },
+      {10, "Q, Z" }
+   };
+
+The *keys* of ``oldPointStructure`` are the Scrabble points stored as ints, and the
+*values* a string of letters. All letters in the array have the Scrabble
```

Comment:
the values are* a string of letters ...

Reply:
in this section the string of letters is later referred to as an array - can you change this to string?